gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -751,8 +775,8 @@ function SWEP:SecondaryAttack()
 
 	local bNotIronsights = not self:GetIronsights()
 
-	self:SetIronsights(bNotIronsights)
 	self:SetZoom(bNotIronsights)
+	self:SetIronsights(bNotIronsights)
```

Comment:
I have to test this again. Might be irrelevant by now. But at some point it was relevant because of the way I have to hack into the FOV stuff. I will test this and report back